scraping/scraping/start_scripts/zalando_de.py

```diff
@@ -34,34 +34,54 @@ def combine_results(
+        "/herrenbekleidung-jacken/": ProductCategory.JACKET.value,
+        "/herrenbekleidung-maentel/": ProductCategory.JACKET.value,
+        "/herrenbekleidung-anzuege/": ProductCategory.SUIT.value,
+        "/herrenbekleidung-jeans/": ProductCategory.JEANS.value,
+        "/herrenbekleidung-hosen/": ProductCategory.PANTS.value,
+        "/herrenbekleidung-hosen-shorts/": ProductCategory.SHORTS.value,
+        "/herrenbekleidung-trainingsanzuege-jogger/": ProductCategory.TRACKSUIT.value,
+        "/unterhosen/": ProductCategory.UNDERWEAR.value,
+        "/unterhemd-herren/": ProductCategory.UNDERWEAR.value,
+        "/herrenbekleidung-struempfe-socken/": ProductCategory.SOCKS.value,
+        "/herrenbekleidung-bademaentel/": ProductCategory.NIGHTWEAR.value,
+        "/herrenbekleidung-nachtwaesche/": ProductCategory.NIGHTWEAR.value,
+        "/herrenbekleidung-bademode/": ProductCategory.SWIMWEAR.value,
+        "/sports-herren-shirts-funktion/": (ProductCategory.TSHIRT.value, {"type": "SPORT"}),
+        "/sports-tanktops-herren/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-poloshirts-herren/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-langarmshirts-herren/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-herren-hemden/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-herren-shirts-trikot/": (ProductCategory.SHIRT.value, {"type": "SPORT"}),
+        "/sports-herren-jacken/": (ProductCategory.JACKET.value, {"type": "SPORT"}),
+        "/sporthosen-herren/": (ProductCategory.PANTS.value, {"type": "SPORT"}),
+        "/sports-herren-pullover-sweater/": (ProductCategory.SWEATER.value, {"type": "SPORT"}),
+        "/trainingsanzug-herren/": (ProductCategory.TRACKSUIT.value, {"type": "SPORT"}),
+        "/funktionsunterwaesche-herren/": (ProductCategory.UNDERWEAR.value, {"type": "SPORT"}),
+        "/sports-herren-struempfe/": (ProductCategory.SOCKS.value, {"type": "SPORT"}),
+        "/sports-bademode-herren/": (ProductCategory.SWIMWEAR.value, {"type": "SPORT"}),
+        "/sportschuhe-herren/": (ProductCategory.SHOES.value, {"type": "SPORT"}),
+        "/sports-herren-rucksack/": (ProductCategory.BACKPACK.value, {"type": "SPORT"}),
+        "/sports-herren-taschen/": (ProductCategory.BAG.value, {"type": "SPORT"}),
+        "/herrenschuhe-sneaker/": ProductCategory.SNEAKERS.value,
+        "/herrenschuhe-schnuerer/": ProductCategory.SHOES.value,
+        "/herrenschuhe-halbschuhe/": ProductCategory.SHOES.value,
+        "/herrenschuhe-business-schuhe/": ProductCategory.SHOES.value,
+        "/herrenschuhe-offene-schuhe/": ProductCategory.SHOES.value,
+        "/herrenschuhe-boots/": ProductCategory.SHOES.value,
+        "/herrenschuhe-hausschuhe/": ProductCategory.SHOES.value,
+        "/taschen-accessoires-taschen-herren/": ProductCategory.BAG.value,
+        "/rucksaecke-herren/": ProductCategory.BACKPACK.value,
```

Comment:
I would prefer option 1, but this can sometimes result in more work for doing the assignment if there are a lot of subcategories. So if we would agree on some standard like described in option 2 then this could be simplified. Might be good to discuss this once again.

Reply:
Added issue about category documentation: https://github.com/calgo-lab/green-db/issues/87